File: CS50x/4. Memory/11.c
/*

> Date Created: July 29, 2023
> Author: Ishaan Rastogi
> Purpose: To swap values of 2 variables
> Operating System: This is only for Windows OS, it may or may not work on other OS
> Program Status: 100% Working

*/

// Garbage values- It means that a variable that you might have defined or declared, if you don't give it an explicit value, who knows what's going to be there.

#include<stdio.h>

void swap ( int *a, int *b);

int main()
{
    int x = 1;
    int y = 2;

    printf("x is %i, y is %i\n", x, y);
    swap (&x, &y); // Calling the swap function and & to get address of x & y
    printf("x is %i, y is %i\n", x, y);
}

void swap ( int *a, int *b)
{
    int tmp = *a;
    *a = *b;
    *b = tmp;
}

/* 

Ctrl + Shift + `

Terminal Code Loading...

gcc 11.c
./a.exe

*/

File: Web Development Projects/15 Intermediate JS/11.js
/*
> Date Created: June 10, 2024
> Author: Ishaan Rastogi
> Purpose: Write lyrics of the song 99 bottles of Beer.
> Operating System: This is only for Windows OS, it may or may not work on other OS
> Program Status: 100% Working
*/

var numberOfBottles = 99
while (numberOfBottles >= 0) {
    var bottleWord = "bottle";
    if (numberOfBottles === 1) {
        bottleWord = "bottles";
    } 
    console.log(numberOfBottles + " " + bottleWord + " of beer on the wall");
    console.log(numberOfBottles + " " + bottleWord + " of beer,");
    console.log("Take one down, pass it around,");
	numberOfBottles--;
    console.log(numberOfBottles + " " + bottleWord + " of beer on the wall.");
}

/* 
For Terminal Code... Ctrl + Shift + `
*/